src/components/internal/convert.ts

```diff
@@ -0,0 +1,123 @@
+import {html} from 'lit';
+import {customElement, state} from 'lit/decorators.js';
+import {InternalElement} from './InternalElement';
+import {fiat} from '@getalby/lightning-tools';
+import {withTwind} from '../twind/withTwind.js';
+import {classes} from '../css/classes.js';
+
+@customElement('satoshi-converter')
+export class SatoshiConverter extends withTwind()(InternalElement) {
+  @state() satoshi = 0;
+  @state() selectedCurrency = 'sat';
+  @state() convertedValue = 0;
+  @state() isSelectVisible = false;
+
+  private localStorageKey = 'selectedCurrency';
+
+  override async firstUpdated() {
```

Comment:
I think we need to store the currency in the zustand store because it is used across multiple independent components

See `src/state/store.ts`